### ISSUE TITLE
The upload target URL you are trying to access has expired. Please request a fresh one. (Rmega::ServerError)

### ISSUE THREAD

**Issue by Souravgoswami** [topac/rmega#30]
If I just do
```ruby
require 'rmega'
storage = Rmega.login('valid_username', 'valid_password')
```
I get:
```
Traceback (most recent call last):
	5: from rmega.rb:2:in `<main>'
	4: from /home/sourav/.gem/ruby/2.7.0/gems/rmega-0.2.6/lib/rmega.rb:34:in `login'
	3: from /home/sourav/.gem/ruby/2.7.0/gems/rmega-0.2.6/lib/rmega/session.rb:94:in `login'
	2: from /home/sourav/.gem/ruby/2.7.0/gems/rmega-0.2.6/lib/rmega/session.rb:144:in `request'
	1: from /home/sourav/.gem/ruby/2.7.0/gems/rmega-0.2.6/lib/rmega/net.rb:7:in `survive'
/home/sourav/.gem/ruby/2.7.0/gems/rmega-0.2.6/lib/rmega/session.rb:150:in `block in request': The upload target URL you are trying to access has expired. Please request a fresh one. (Rmega::ServerError)
```

**Comment by topac** [comment]
That's unusual, since if the user/pass is wrong you should get `Object (typically, node or user) not found (Rmega::ServerError)`. Have you tried with a different account or from a different IP address?

**Comment by Souravgoswami** [comment]
Hi, generally my computers use my phone's internet via hotspot, so I am not sure about trying it from different IP addresses.

I do have another mega account I have
+ Confirmed that the email and password are valid.
+ Used the username instead of email.

But that didn't work with that another account either.

If I put wrong credential, the error raised is same, `Rmega::Server error`

**Comment by Aravind-Belagaje** [comment]
Even i am facing same may be related to https://github.com/rclone/rclone/issues/2771

**Comment by topac** [comment]
Thanks @Aravind-Belagaje, a new version of the gem will be released soon in order to support account ver.2 (this will solve @Souravgoswami problem)

**Comment by topac** [comment]
fixed in rmega 0.2.7
